src/test/groovy/org/araneforseti/boundary/fields/FieldTest.groovy

```diff
@@ -18,15 +18,15 @@ class FieldTest {
         assert !scenarios_contains_value(null, new TestField("testField", "foo", false))
     }
 
-    @Test
-    void field_can_have_message_name() {
-        assert (new TestField("testField", "foo", false, "message")).messageName == "message"
-    }
-
-    @Test
-    void field_message_name_defaults_to_name() {
-        assert (new TestField("testField", "foo", false)).messageName == "testField"
-    }
+//    @Test
```

Comment:
If you are commenting out tests, they should probably just be deleted. 

Additionally, I think these tests are still worth having around, just in a different form